src/main/java/mate/academy/onlinebookstore/model/Book.java

```diff
@@ -0,0 +1,40 @@
+package mate.academy.onlinebookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name = "books")
+@Getter
+@Setter
+public class Book {
+    @Column(nullable = false)
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
You don't need this annotation for `id` field, because value for this field is generated by the database when Hibernate stores your entity. I suggest to read about `GenerationType` strategies in Hibernate
```suggestion
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
```